temporal-sdk/src/main/java/io/temporal/client/WorkflowClientOptions.java

```diff
@@ -33,6 +33,7 @@
 
   private static final WorkflowClientOptions DEFAULT_INSTANCE;
   private static final String DEFAULT_NAMESPACE = "default";
+  private static final String DEFAULT_BINARY_CHECKSUM = "none";
```

Comment:
Why not an empty string as it is now?

Reply:
Having some value could potentially be useful in case when someone just added their own binary checksum and wants to roll back.
I'm okay changing it to empty string if you prefer it.

Reply:
Done